Library/Formula/ipfs.rb

```diff
@@ -0,0 +1,47 @@
+require "language/go"
+
+class Ipfs < Formula
+  desc "IPFS is The Permanent Web - A new peer-to-peer hypermedia protocol"
+  homepage "http://ipfs.io/"
+  url "https://github.com/ipfs/go-ipfs.git",
+    :tag => "v0.3.5",
+    :revision => "952dc9c60fdff27902749222fdc30164e7eea1ee"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/tools/godep" do
+    url "https://github.com/tools/godep.git",
+      :revision => "58d90f262c13357d3203e67a33c6f7a9382f9223"
+  end
+
+  go_resource "github.com/kr/fs" do
+    url "https://github.com/kr/fs.git",
+      :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
+  end
+
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools",
+      :using => :git,
+      :revision => "b1aed1a596ad02d2aa2eb5c5af431a7ba2f6afc4"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/ipfs/"
+    ln_sf buildpath, buildpath/"src/github.com/ipfs/go-ipfs"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/tools/godep" do
+      system "go", "install"
+    end
+
+    cd "cmd/ipfs" do
+      system "make", "build"
+      bin.install "ipfs"
+    end
+  end
+
+  test do
+    system "false"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!